autoload/sy/repo.vim

```diff
@@ -257,7 +257,7 @@ function! sy#repo#diffmode(do_tab) abort
   try
     execute chdir fnameescape(b:sy.info.dir)
     leftabove vnew
-    if has('iconv')
+    if (fenc != &enc) && has('iconv')
```

Comment:
Typo: `fenc` :>

Reply:
No it is not a typo.
That is for  #278.
In that case, `fenc` is a local variable and must be kept before `tabedit` and `vnew`.

Reply:
Oops, I stand corrected. Sorry for the noise.